module2/misc/deploying_with_render.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: page
+title: Using Render.com to Deploy a Rails App (Student Guide)
+---
+<section class="note">
+<p>These instructions were compiled from <a href="https://render.com/docs/deploy-rails" target="_blank">Render.com</a>'s existing documentation and resources. You are free to ignore this page and follow directions on Render's website to deploy your application, if you wish. The instructions and links below have been tested as of March 2023.</p>
+
+<p>Turing is not associated or affiliated with Render.com in any way; this page was created as a reference for students to deploy their applications for the first time. 
+</p>
+</section>
+
+## When should I deploy?
+- Deploy to Render (or more generally, the _production_ environment) when your application has reached a new apex in feature availability - in other words, when your `main` branch has been updated to where you are confident that you want it to be available on the internet. 
+- When deploying, Render will take ___your remote copy___, you'll be able to select which branch (we recommend `main`), and deploy it to your server. So, follow good git practice and only merge to `main` once your team has decided the code quality is good enough to deploy a new version of the app. 
+
+## Getting Started
+1. Create an account on Render, if you haven't already. (You can use GitHub to create an account to make your sign-in even easier.)
+
+2. Locally in the repository you'll be deploying, update the secret key before deploying: 
+   1. Delete the existing `config/credentials.yml.enc` file. 
+   2. In Terminal in this repository, run `rails credentials:edit`.
+         - You may need to specify `EDITOR="code" bin/rails credentials:edit` for it to work, otherwise it may give you errors. When successful, it will generate 2 files called `credentials.yml.enc` and `master.key`. The `master.key` file will be ignored by git - this is normal, you do not want to share this key with other teammates. 
+   3. Commit and push the change that added the new `credentials.yml.enc` file to your remote. 
+
+## Deploying an Existing Rails App
+
+### Create DB Service
```

Comment:
I like that this is the first step! When I deployed I set up the web service first and had to go back and forth to copy over the Internal DB URL, this feels smoother.